mem0/memory/main.py

```diff
@@ -159,6 +165,14 @@ def add(
                     {"memory_id": function_result, "function_name": function_name},
                 )
         capture_event("mem0.add", self)
+
+        if self.version == "v1.1" and self.enable_graph:
```

Comment:
Here version is `v1.1` but here `mem0/configs/base.py` version is defined as `v1`. This seems odd, is this expected behaviour because?

Reply:
v1.1. has graphrag and 1.0 is the current one. This is set such that users can use OSS without breaking the return datatype of search query. Because right now we are returning a list but if we are using graphRAG then we need to return a dict with two keys.

Reply:
Alright, makes sense. Thanks.